### ISSUE TITLE
5.0.1-alpha2 ListHistoricalTradesAsync.TimestampUTC returned as local time marked as kind utc

### ISSUE THREAD

**Issue by jeffwlandry** [alpacahq/alpaca-trade-api-csharp#295]
iHistoricalTrade.TimeUtc seems to be US East Coast time, but the DateTime "kind" is marked utc.

**Comment by OlegRa** [comment]
@jeffwlandry OK, I've checked the raw data received from the Alpaca REST endpoint, and looks like it's totally wrong. They send EST timestamps formatted as UTC timestamps (with the `Z` time zone specifier). Of course, I can convert these wrong dates into the correct ones inside SDK but I totally dislike this approach - if in the future Alpaca will fix this problem on the server (and I hope they will do it soon) I have to roll back these changes in SDK.

I've already reported this problem to Alpaca guys using my internal channel but if you will open this question via the standard Alpaca support channels (forum, Slack, and email) I think it will also help to resolve this issue faster. I'm not sure about another SDK but most probably if they parse the RFC3339 date-time string properly they experience the same problem.
